### PR TITLE
sbt 1.0.x preparation: bump dependencies

### PR DESCRIPTION
**Pull request by larsrh**
Should probably be merged before #102.

A release of `laika-sbt` for 1.0.x is still pending, but it's getting closer (planet42/Laika#59).

### CODE REVIEW

**Comment by larsrh** [comment]
There is one failed test in for Laika: 

```
/tmp/sbt_f0e2960e/blog-post/project/CustomDirectives.scala:27: not found: value context

     context.map { context =>
```

@jenshalm Could you maybe take a look? This is caused by an API change from Laika 0.6.0 to 0.7.0.

**Comment by jenshalm** [comment]
@larsrh: Just a quick shot (without running it):

```scala
    cursor.map { cursor =>
      val posts = cursor.root.children.collect {
        case pd: TreeCursor =>
          pd.children.map {
            case d: DocumentCursor =>
            val path = d.target.path
            val refPath = cursor.parent.target.path
            PostsTocElement(titleOrName(d.target),
              by(d.config),
              PathInfo.fromPath(path, refPath.parent),
              Styles("toc"))
          }
      }.flatten
```

**Comment by larsrh** [comment]
@jenshalm Thanks, updated.

File: src/sbt-test/laika/blog-post/project/CustomDirectives.scala [hunk 1]
```diff
@@ -24,16 +25,17 @@ object CustomDirectives extends StandardDirectives {
...
-            val path = d.path
-            val refPath = context.parent.path
-            PostsTocElement(titleOrName(d),
-              by(d.config),
-              PathInfo.fromPath(path, refPath.parent),
-              Styles("toc"))
+    cursor.map { cursor =>
+      val posts = cursor.root.children.collect {
+        case pd: TreeCursor =>
+          pd.children.map {
```
Comment (jenshalm) [inline src/sbt-test/laika/blog-post/project/CustomDirectives.scala]: Sorry, I corrected my comment afterwards. `collect` is safer here as the above might cause match errors.

Comment (larsrh) [inline src/sbt-test/laika/blog-post/project/CustomDirectives.scala]: Fixed.

**Pull request merged**